pulsar-sql/presto-pulsar/pom.xml

```diff
@@ -109,6 +109,32 @@
           <version>${joda.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.prestosql</groupId>
+            <artifactId>presto-record-decoder</artifactId>
+            <version>${presto.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>pulsar-client-original</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Does this dependency is necessary?

Reply:
Yes , `PulsarRowDecoder.decodeRow()` call `GenericXXXSchema.decode(ByteBuf byteBuf) ` for GenericRecord.

Reply:
It seems that the dependency `pulsar-client-original` is already in the `pulsar-client-admin-original`.